src/NuGet.Config

```diff
@@ -4,8 +4,11 @@
     <add key="enabled" value="True" />
   </packageRestore>
   <packageSources>
-    <add key="myget.org dotnet-buildtools" value="https://www.myget.org/F/dotnet-buildtools/" />
     <add key="myget.org dotnet-corefx" value="https://www.myget.org/F/dotnet-corefx/" />
+    <add key="myget.org dotnet-buildtools" value="https://www.myget.org/F/dotnet-buildtools/" />
     <add key="nuget.org" value="https://www.nuget.org/api/v2/" />
   </packageSources>
+  <activePackageSource>
+    <add key="All" value="(Aggregate source)" />
+  </activePackageSource>
```

Comment:
It was necessary because although multiple package sources were specified, on my machine it was not searching all of them.  From the docs (http://docs.nuget.org/docs/reference/nuget-config-settings) "(Aggregate source)" is a special value that means every specified package source is active.